src/Blocks/components/sorting/sorting-admin.scss

```diff
@@ -0,0 +1,35 @@
+.es-sorting {
+	&__update {
+		display: none;
+	}
+
+	&__trigger,
+	&__update {
+		&.es-submit {
+			width: auto;
+		}
+	}
+
+	> * {
+		> * {
+			transition: {
+				property: border, padding, margin;
+				timing-function: ease-out;
+				duration: 0.3s;
+			}
+		}
+	}
+
+	> .is-active {
+		> * {
+			cursor: ns-resize;
+			border: 2px dashed var(--global-colors-es-french);
+			padding: 20px;
```

Comment:
How come this is in `px`, and not in `rem`?

Reply:
this is only for admin part /settings so there is no need for rems because WP admin doesn't use rems at this point. And we need to rewrite all our stiles in rems because it is currently 50/50